### ISSUE TITLE
Vscode theme with nuance

### ISSUE THREAD

**Issue by backseats** [kepano/flexoki#29]
![image](https://github.com/kepano/flexoki/assets/87050725/7cbe3ba4-46cc-49c1-a545-50b8d958ae45)

It would be great to use the light and dark tones provided in the theme for some more nuance similar to how Eppz! does it

I can add this later but wanted to throw it out if anyone wanted to get started on it

**Comment by Railly** [comment]
I agree with your point. When I was developing on the theme, sticking to a consistent set of token color guidelines was difficult.

I've already got an algorithm that maps keywords based on token abbreviation (tx, tx-2, ui, etc), but I could create a similar one for mapping over constants, keywords, strings, and so on. Even my own theme could benefit from it, so it's a win-win.

I can create a new repo only focused only on flexoki nuances, based on color rules (such Eppzz! image) to be easily customizable. 

What do you think @kepano? 

Options are:
- New isolated repo based on clear code guidelines, based on how Eppzz! do. If so, who gets the ownership?
- Continue updating my repo with flekoki variants, as well as yours in this repo.
- No action, no variants, just one single variation.

**Comment by backseats** [comment]
One note kepano made to me on Twitter around opacities, I wanted to call out here

https://x.com/kepano/status/1711016436284445174?s=46

**Comment by kepano** [comment]
I like this idea. I think it makes sense. I have been working on [my port for Sublime Text](https://github.com/kepano/flexoki-sublime) and was running into similar issues.

As @backseats just mentioned this will require me too add more color variations if we want to use opacity relationships — for the reasons explained in the Flexoki documentation. The muted/faint versions of the colors will not be derived using opacity, instead they will be fainter but still opaque shades of colors that are more saturated.

I think the token guidelines should be in this repo. I am open minded about where the generator code should be located. It would be neat if it can work for multiple syntax highlighters (e.g. VSCode, Sublime Text, PrismJS, etc), but that might be too ambitious.

**Comment by Railly** [comment]
I've just seen the generator folder. I'll be experimenting for a while on my own, then I'll come up with updates.

**Comment by Railly** [comment]
Hey, new updates here. @kepano 

I've launched a new repo with 11 generators at the moment. I've replicated One Hunter Theme and Flexoki variants with it. There is an extensive README there, if you have any questions, you can leave it here. =)

https://github.com/Railly/tinte

**Comment by ScottKillen** [comment]
[Mixbox](https://scrtwpns.com/mixbox/) might be good for this project.
